src/transformers/models/deberta_v2/tokenization_deberta_v2.py

```diff
@@ -325,16 +326,7 @@ def __setstate__(self, d):
         self.spm.Load(self.vocab_file)
 
     def tokenize(self, text):
-        pieces = self._encode_as_pieces(text)
-
-        def _norm(x):
-            if x not in self.vocab or x == "<unk>":
-                return "[UNK]"
-            else:
-                return x
-
-        pieces = [_norm(p) for p in pieces]
-        return pieces
+        return self._encode_as_pieces(text)
```

Comment:
I haven't followed all the development on this PR, so flagging this change to make sure it's a bug fix and not an unintentional breaking change.

Reply:
Thanks for pointing this out! 

This part was discussed in the first version of the PR #14928 ([here](https://github.com/huggingface/transformers/pull/14928#pullrequestreview-842924296)) and with @patrickvonplaten ([here](https://huggingface.slack.com/archives/C01N44FJDHT/p1640712012260200)). TLDR, we saw it as a bug for this tokenizer.

This relates more generally to a problem of consistency in the `tokenize` method between the different types of tokenizer (cf issue #15138)

Reply:
This relates more generally to a problem of consistency in the `tokenize` method between the different types of tokenizer (cf issue #15138). I think it is the good place for me to say that personally, I think the tokenize method should always return something equivalent to `tokenizer.convert_ids_to_tokens(tokenizer.encode(text))`, but this is not the case for all unigram-based tokenizers (it's like this inside the `tokenizers` lib and a lot of tokenizers are in this situation). Here this change will make `ŧokenize` not equivalent to `tokenizer.convert_ids_to_tokens(tokenizer.encode(text))`.

If you think that making this change is a mistake, please share! :smile:

Reply:
I just wanted the full context since there was no comment around it in this PR. LGTM!